lib/hawkular/inventory/entities.rb

```diff
@@ -0,0 +1,217 @@
+# It contains class definitions that are used by the inventory REST client
+module Hawkular::Inventory
+  # A Basic inventory entity with id, name, path and optional properties
+  class BaseEntity
+    # @return [String] Full path of the entity
+    attr_reader :path
+    # @return [String] Name of the entity
+    attr_reader :name
+    # @return [String] Name of the entity
```

Comment:
this should be id and not Name in the comment (a copy paste from line 7 :smile) but is there really a need for comments for things like name/id? what is a `@` sign in the comment for?